tests/ci_build/conda_env/linux_cpu_test.yml

```diff
@@ -14,7 +14,7 @@ dependencies:
 - pylint
 - numpy
 - scipy
-- scikit-learn
+- scikit-learn>=1.4.1
```

Comment:
Should we also add `scikit-learn>=1.4.1` to `pyproject.toml`?

Reply:
No need, for older versions, the code has no effect.